Update recommended-settings-for-eop-and-office365-atp.md (#987)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md

This topic describes these Microsoft-recommended settings to help protect your Office 365 users.

> [!TIP]

> > There is a new PowerShell Module that you can download called the Office 365 Advanced Threat Protection Recommended Configuration Analyzer (ORCA) that helps determine some of these settings. When run as an admin in your tenant, Get-ORCAReport will help generate an assessment of the anti-spam, anti-phish, and other message hygiene settings. You can download this module at https://www.powershellgallery.com/packages/ORCA/.

Reviewer: ```suggestion
> There is a new PowerShell module called the Office 365 Advanced Threat Protection Recommended Configuration Analyzer (ORCA) that you can get to determine some of these settings. When run as an admin in your tenant, the `Get-ORCAReport` cmdlet generates an assessment of the anti-spam, anti-phish, and other message hygiene settings. You can download this module from the [PowerShell Gallery](https://www.powershellgallery.com/packages/ORCA).
```
